pkg/sql/parser/sql.y

```diff
@@ -6280,6 +6283,17 @@ show_enums_stmt:
 }
 | SHOW ENUMS error // SHOW HELP: SHOW ENUMS
 
+// %Help: SHOW FUNCTION - Display function information.
+// %Category: Misc
+// %Text: SHOW FUNCTION <function_name>
+// %SeeAlso: WEBDOCS/show-function.html
+show_function_stmt:
+  SHOW FUNCTION func_name_no_crdb_extra
```

Comment:
nit: I'm actually prefer to use `db_object_name` instead, `func_name_no_crdb_extra` has a `prefixed_column_path` which can have 4 parts in the qualified name which can be confusing.

Reply:
Is a function name an "object name" though? Don't we want a `ResolvableFunctionReference` so that the function can be resolved? I would have used `func_name` instead of `func_nam_no_crdb_extra`, but it was causing reduce/reduce conflicts.

Reply:
I've updated this to use `db_object_name`.